packages/ui/src/App/context.tsx

```diff
@@ -15,26 +23,21 @@ const ThemeProvider =
 const MemoryRouter =
   ReactRouterDOM != null ? ReactRouterDOM.MemoryRouter : React.Fragment;
 
-const findExamplesForComponent = (c: Component) => {
-  const exampleFileExport = userImport.examples.filter(
-    (ex: any) => ex.default !== null && ex.default === c.component,
+const exampleImports: ExampleImport[] = userImport.examples;
+
+const findExamplesForComponent = (c: Component): Example[] => {
+  const exampleFileExport = exampleImports.filter(
+    ex => ex.forComponent !== null && ex.forComponent === c.component,
```

Comment:
Should maybe examples with null properties be filtered out earlier on? would an example with null forComponent ever be shown?

Reply:
Oops, this should be `!= null` instead. The `forComponent` can be undefined if the user does not have a default export from the examples file.